qancode/clickpaths.py

```diff
@@ -485,3 +485,21 @@ def perform_action(self):
                 self.filenames.append(filename)
                 self.download_start_times.append(download_start_time)
                 time.sleep(2)
+
+
+class ClickSearchResultItem:
+    """
+    Clicks the first item on a page of search results.
+    """
+
+    def __init__(self, driver):
+        self.driver = driver
+        self.perform_action()
+
+    def perform_action(self):
+        try:
+            item_link = self.driver.find_element_by_xpath(SearchPageList.search_result_item)
```

Comment:
How do we get to the search page in the first place?

Reply:
We give QANCODE an item_type going directly to a search page, like ('/search/?type=AntibodyLot&status=in+progress&sort=date_created', ClickSearchResultItem)